soccer/motion/MotionControl.cpp

```diff
@@ -178,45 +175,40 @@ void MotionControl::run() {
 
 
 		//	evaluate path - where should we be right now?
-		Point targetPos;
-		bool pathValidNow = _robot->path()->evaluate(
-			timeIntoPath,
-			targetPos,
-			targetVel);
+		bool pathValidNow = _robot->path()->evaluate(timeIntoPath, target);
 		if (!pathValidNow) {
-			targetVel.x = 0;
-			targetVel.y = 0;
+			target.vel = Geometry2d::Point();
 		}
 		//	tracking error
-		Point posError = targetPos - _robot->pos;
+		Point posError = target.pos - _robot->pos;
 
 		//	acceleration factor
-		Point nextTargetVel, _;
-		_robot->path()->evaluate(timeIntoPath + 1.0/60.0, _, nextTargetVel);
-		Point acceleration = (nextTargetVel - targetVel) / 60.0f;
+		Planning::MotionInstant nextTarget;
+		_robot->path()->evaluate(timeIntoPath + 1.0 / 60.0, nextTarget);
+		Point acceleration = (nextTarget.vel - target.vel) / 60.0f;
```

Comment:
Do we need to rotate nextTarget.vel before doing this?  You have it the same as it was before, but I think it should have been rotated then too...